front/src/App.tsx

```diff
@@ -24,23 +24,31 @@ import { AlertWithoutAction } from "components/Alerts";
 import { ErrorBoundary } from "@sentry/react";
 import Dashboard from "@boxtribute/shared-components/statviz/dashboard/Dashboard";
 
-interface IProtectedRouteProps {
+type ProtectedRouteProps = {
```

Comment:
is there a reason you changed this styling format? Hope you will update it then in the README

Reply:
May I push for this style from now onwards? I'll change the README if I have your blessing. 😁